packages/patternfly-4/react-core/src/components/Dropdown/Dropdown.js

```diff
@@ -37,11 +38,7 @@ const propTypes = {
 };
 
 const defaultProps = {
-  id:
-    new Date().getTime() +
-    Math.random()
-      .toString(36)
-      .slice(2),
+  id: getUniqueId(),
```

Comment:
this does not work. Default props are only instantiated once on when this variable , `defaultProps` is created. The id will need to be setup in a "constructor" or better as a class property
```jsx
class Something extends React.Component {
  this.id = this.props.id || getUniqueId();
}
```